plugin/kubernetes/kubernetes.go

```diff
@@ -192,6 +194,7 @@ func (k *Kubernetes) getClientConfig() (*rest.Config, error) {
 			return nil, err
 		}
 		cc.ContentType = "application/vnd.kubernetes.protobuf"
+		cc.UserAgent = fmt.Sprintf("%s/%s-%s (%s/%s)", coremain.CoreName, coremain.CoreVersion, coremain.GitCommit, runtime.GOOS, runtime.GOARCH)
```

Comment:
I'm not sure I like the version concatenated with the git commit. I would prefer a space separator there.

Maybe:
```suggestion
		cc.UserAgent = fmt.Sprintf("%s/%s git_commit:%s (%s/%s)", coremain.CoreName, coremain.CoreVersion, coremain.GitCommit, runtime.GOOS, runtime.GOARCH)
```

Reply:
``` 
./coremain/run.go:                      appVersion = fmt.Sprintf("%s (+%s %s)", strings.TrimPrefix(gitNearestTag, "v"), GitCommit, buildDate)
```
Should we try to make it a short commit format?

Reply:
Long hash is fine IMO.